integration/setup/setup.go

```diff
@@ -60,13 +59,6 @@ func installResources(ctx context.Context, config Config) error {
 		}
 	}
 
-	{
```

Comment:
We are installing chart-operator in the resource level, so don't need to care it from e2e setup.